2022/Day2/Day2.py

```diff
@@ -0,0 +1,44 @@
+file2 = open('input.txt', 'r')
+
+total_score = 0
+
+while True:
+    Line = file2.readline().strip()
```

Comment:
Please don't name variable with a capital letter. The convention in Python is to name variables in all_lowercase_with_underscores, commonly known as snake_case.

Reply:
Fixed.